packages/ingest/package.json

```diff
@@ -23,10 +27,11 @@
     "CUMULUS",
     "NASA"
   ],
-  "homepage": "https://github.com/nasa/cumulus/tree/master/packages/ingest",
+  "homepage": "https://github.com/nasa/cumulus/tree/master/packages/ingest#readme",
   "repository": {
     "type": "git",
-    "url": "https://github.com/nasa/cumulus"
+    "url": "https://github.com/nasa/cumulus",
+    "directory": "packages/ingest"
```

Comment:
What does this do - take you directly to this folder on the repo when you run `npm repo` or something?

Reply:
That's what I would expect from [the docs](https://docs.npmjs.com/files/package.json#repository), but I haven't actually seen it have any effect 🤷‍♂️